README.md

```diff
@@ -3,23 +3,19 @@ Highly Agile Masks Made Effortlessly from RTL
 
 This is the master (unstable) development branch.
 
-Stable numbered releases can be found below:
-
-Releases: https://github.com/ucb-bar/hammer/releases
-
 Documentation
 =============
-Here are links to various pieces of documentation currently available:
+The main documentation is found at https://hammer-vlsi.readthedocs.io
```

Comment:
Should also link to the local `doc/` dir

Reply:
I disagree. `doc/` is just the sources for the readthedocs, so it's all the same info but harder to read...

Reply:
Hmm, good point. Maybe say something like sources are available in `doc/` for offline reading or something?